product_docs/docs/migration_portal/3.4.0/index.mdx

```diff
@@ -19,13 +19,24 @@ legacyRedirectsGenerated:
 
 <div id="index" class="registered_link"></div>
 
-Migration Portal is a web-based service for migrating Oracle database schemas to the EDB Postgres platform. The Migration Portal assesses and analyzes Oracle database schemas and converts types, tables, sequences, constraints, triggers, views, stored procedures, packages, database links (dblinks), materialized views, and indexes, producing DDLs compatible with EDB Postgres Advanced Server.
+Migration Portal is a web-based service for migrating Oracle database schemas to the EDB Postgres Advanced Server platform. The Migration Portal assesses and analyzes Oracle database schemas and converts:
+-  Types
+-  Tables
+-  Sequences
+-  Constraints
+-  Triggers
+-  Views
+-  Stored procedures
+-  Packages
+-  Database links (dblinks)
+-  Materialized views
+-  Indexes
+
+The migration produces DDLs compatible with EDB Postgres Advanced Server.
 
 The user-friendly portal interface simplifies the assessment and migration process. Log in to the portal and start with the migration process.
 
-The Migration Portal guide provides a high-level description of the steps involved in the migration process. The guide also includes solutions to common migration problems and details of unsupported features and their potential workarounds.
-
-EnterpriseDB has helped companies migrate their existing database systems to Postgres for years. For more information, visit the EnterpriseDB website at: <https://www.enterprisedb.com/>
+EnterpriseDB has helped companies migrate their existing database systems to Postgres for years. For more information, see the (EnterpriseDB website)[https://www.enterprisedb.com/].
```

Comment:
Fixed the broken link here while i was working on something else.

Reply:
Thanks.